pkg/agent/openflow/multicast.go

```diff
@@ -147,3 +185,34 @@ func (f *featureMulticast) replayGroups() {
 		return true
 	})
 }
+
+func (f *featureMulticast) multicastToRemoteFlows(groupID binding.GroupIDType, firstMulticastTable binding.Table) []binding.Flow {
+	return []binding.Flow{
+		// This flow outputs the IGMP report message sent from Antrea Agent to an OpenFlow group which is expected to
+		// broadcast to all the other Nodes in the cluster. The multicast groups in side the IGMP report message
+		// include the ones local Pods have joined in.
+		MulticastRoutingTable.ofTable.BuildFlow(priorityHigh).
+			Cookie(f.cookieAllocator.Request(f.category).Raw()).
+			MatchProtocol(binding.ProtocolIGMP).
+			MatchInPort(openflow13.P_CONTROLLER).
+			Action().LoadRegMark(CustomReasonIGMPRegMark).
+			Action().Group(groupID).
+			Done(),
+		// This flow ensures the IGMP report message sent from Antrea Agent to bypass the check in SpoofGuardTable.
+		ClassifierTable.ofTable.BuildFlow(priorityNormal).
+			Cookie(f.cookieAllocator.Request(f.category).Raw()).
+			MatchInPort(openflow13.P_CONTROLLER).
+			Action().GotoTable(SpoofGuardTable.GetNext()).
+			Done(),
+		// This flow ensures the multicast packet sent from a different Node via the tunnel port to enter Multicast
+		// pipeline.
+		ClassifierTable.ofTable.BuildFlow(priorityHigh).
```

Comment:
this has nothing to do the groupID and "toRemote", right? shouldn't it be in `initFlows`?